oidc-api/build.gradle

```diff
@@ -42,10 +44,25 @@ test {
     exclude 'uk/gov/di/authentication/oidc/contract/**'
 }
 
-task contractTest (type: Test) {
+var pactDir = "$rootDir/oidc-api/src/test/pact"
+
+task pactConsumerTests (type: Test, group: "verification") {
     useJUnitPlatform()
     include '/uk/gov/di/authentication/oidc/contract/**'
-    systemProperties['pact.rootDir'] = "$rootDir/oidc-api/src/test/pact"
+    systemProperties['pact.rootDir'] = pactDir
+}
+
+pact {
+    publish {
+        pactDirectory = pactDir
+        pactBrokerUrl = "$System.env.PACT_URL".toString() + "?testSource=" +"$System.env.PACT_BROKER_SOURCE_HEADER".toString()
+        pactBrokerUsername = "$System.env.PACT_USER".toString()
+        pactBrokerPassword = "$System.env.PACT_PASSWORD".toString()
+        consumerVersion = "$System.env.CONSUMER_APP_VERSION".toString()
+        tags = [
```

Comment:
For IPV core we're putting the git branch into the `consumerBranch` property instead. We don't have confirmation yet from Lime that they can use that property yet when pulling down pacts though (they are going to try to get that to work).